src/Promitor.Core.Scraping/Configuration/Serialization/v1/Core/V1Deserializer.cs

```diff
@@ -8,79 +8,43 @@ namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Core
 {
     public class V1Deserializer : Deserializer<MetricsDeclarationV1>
     {
-        private readonly IDeserializer<AzureMetadataV1> _azureMetadataDeserializer;
-        private readonly IDeserializer<MetricDefaultsV1> _defaultsDeserializer;
         private readonly IDeserializer<MetricDefinitionV1> _metricsDeserializer;
 
         public V1Deserializer(IDeserializer<AzureMetadataV1> azureMetadataDeserializer,
```

Comment:
Looks a lot better, thanks :ok_hand:

Reply:
Yeah - I think we can simplify it even more by adding support for deserializing `YamlSequenceNode` objects, but I didn't want to change too much at once (or have to figure out how to do that :smile:).